saleor/dashboard/collection/forms.py

```diff
@@ -45,3 +45,8 @@ class AssignHomepageCollectionForm(forms.ModelForm):
     class Meta:
         model = SiteSettings
         fields = ('homepage_collection',)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['homepage_collection'].queryset = Collection.objects.\
```

Comment:
I think we could break the line after `queryset =` to make it a little bit prettier